app/models/user.rb

```diff
@@ -1,4 +1,10 @@
 class User < ActiveRecord::Base
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
I don't think we should use email for devise.  Just username & password and here is why:

1. Running, managing emails servers is a hassle for me
2. For users running private instances, the same applies
3. We want to know less about our users, not more.  If users want to be anonymous, we should make it easier for them.

One downside is if users lose their password accounts aren't recoverable but I think we can improve this later on.  Also, the user base are intelligent SRE/DevOps/SysAdmins - not end user types.